src/main/java/uk/gov/ch/developer/docs/config/NavBarConfig.java

```diff
@@ -25,25 +25,24 @@
     public NavBarModel getNavBarModel() {
 
         NavBarModel model = new NavBarModel();
-        NavItemList manageApplications = model.addHeading("Manage Applications");
+        NavItemList manageApplications = model
+                .addHeading("Manage Applications", UserRequired.userRequired);
 
```

Comment:
I created this enum because I thought it was ambiguous if true meant it checked for users or if it required users.
Interested if others think it was helpful of just unnecessary.

Reply:
That's sensible.